apps/protailwind/src/pages/tutorials/[module]/[lesson]/exercise.tsx

```diff
@@ -0,0 +1,5 @@
+export {getStaticProps} from './index'
+export {getStaticPaths} from './index'
+import ExercisePage from './index'
+
+export default ExercisePage
```

Comment:
this is responsible for creating `[lesson]/exercise` route. could be called differently, e.g. `repl`, but this felt more general.